src/interface/swagger.json

```diff
@@ -210,7 +210,15 @@
                         "format": "date-time",
                         "description": "The end date for query (relates to central-ledger.settlementWindow.createdDate). Can be used together with `fromDateTime'. eg 2017-07-21T17:32:28Z\n",
                         "required": false
+                    },
+                    {
+                        "in": "query",
+                        "name": "currency",
+                        "type": "string",

```

Comment:
Should this be a currency code like what we have in the other swagger specs?

Reply:
Thanks Lewis. This is a good point, but enforcing currency code by swagger enum or definition across all services should be a separate story as neither it was included in the epic nor the current swagger definition had it defined for other requests where currency string is accepted.

Reply:
Sure. All good.